modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-cloud/pom.mustache

```diff
@@ -99,48 +105,39 @@
             <groupId>com.github.joschi.jackson</groupId>
             <artifactId>jackson-datatype-threetenbp</artifactId>
         {{^parentOverridden}}
-            <version>2.9.10</version>
+            <version>2.12.5</version>
         {{/parentOverridden}}
         </dependency>
         {{/threetenbp}}
         {{#openApiNullable}}
         <dependency>
             <groupId>org.openapitools</groupId>
             <artifactId>jackson-databind-nullable</artifactId>
-            {{^parentOverridden}}
-            <version>0.2.1</version>
-            {{/parentOverridden}}
+        {{^parentOverridden}}
+            <version>0.2.2</version>
+        {{/parentOverridden}}
         </dependency>
         {{/openApiNullable}}
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-test</artifactId>
-            <scope>test</scope>
-        </dependency>
-{{#hateoas}}
-        <!-- Spring HATEOAS -->
+        {{#hateoas}}
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-hateoas</artifactId>
         </dependency>
-{{/hateoas}}
-{{#useBeanValidation}}
+        {{/hateoas}}
+        {{#useBeanValidation}}
```

Comment:
So maybe a general question - I assume by having this match the indentation of the pom file this is causing white space to be generated, which is likely why it was fully left aligned. 

Are there any community guidelines on this?